Dockerfile

```diff
@@ -22,6 +23,9 @@ RUN gem install bundler
 RUN bundle install  --without production
 RUN yarn install
 
+# Remove CMAKE
+RUN apt-get purge -y --auto-remove cmake
```

Comment:
Keep this as this won't save space anyway, also create one more docker layer for no good reason. Could you check if the same is needed for the production docker file?

Reply:
for production this will not required `rugged` is only using by `undercover` gem, so in production environment the gem is not going to be installed